back/engines/free/email_campaigns/app/mailers/email_campaigns/mention_in_official_feedback_mailer.rb

```diff
@@ -29,6 +29,8 @@ def header_message
     end
 
     def preheader
+      # initiating_user_first_name is never present.
+      # TODO: fix. We previously removed this field from the event https://github.com/CitizenLabDotCo/citizenlab/commit/72a778f#diff-f9b809a7dcedf9fd13177f3b9b05ac24ddde8f5f36d197d65e256c20b22ae62aL25
```

Comment:
@sebastienhoorens Do I understand correctly that it wasn't changed by accident and we need to fix it? And the same question about https://github.com/CitizenLabDotCo/citizenlab/pull/6525/files#diff-cb5c0a0078d11891135fa930993fe84f6d470d835c65ad55009e963ae8a46ed7R19

Reply:
@alexander-cit It's a bit vague in my memory, but I think the goal was to clean up the event payload to only include attributes that will be used. If we need it back, feel free to add it back.

At some point we had a separate service (cl2-emails) for emails, and a payload was sent through RabbitMQ (I think). The events sometimes include more info than is necessary, so I think I tried to clean this up a bit.

Reply:
Got it. Thanks for the clear explanation!